src/integration_test.rs

```diff
@@ -139,7 +139,7 @@ fn test_instantiate() {
     assert_ne!(cw20_token.addr(), amm_addr);
 
     let info = get_info(&router, &amm_addr);
-    assert_eq!(info.lp_token_address, "Contract #2".to_string());
+    assert_eq!(info.lp_token_address, "contract2".to_string());
```

Comment:
lets just go ahead and remove this assertion. doesn't actually test anything about the contract's behavior.

Reply:
fixed here : 1b42d76915f6cd3de62186b80b9a368ca836ee10